presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ScalarFromAnnotationsParser.java

```diff
@@ -99,11 +107,38 @@ private static List<ScalarHeaderAndMethods> findScalarsInFunctionSetClass(Class<
         return builder.build();
     }
 
+    private static Optional<ScalarStatsHeader> getScalarStatsHeader(Method annotated)
+    {
+        Optional<ScalarStatsHeader> scalarStatsHeader;
+        ScalarFunctionConstantStats constantStatsAnnotation =
+                annotated.getAnnotation(ScalarFunctionConstantStats.class);
+        List<java.lang.reflect.Parameter> params =
+                Arrays.stream(annotated.getParameters())
```

Comment:
nit: can we just import `Parameter` instead of using `java.lang.reflect.Parameter`?